src/restapi/fIncomingWebhooks.ts

```diff
@@ -16,30 +18,31 @@ export async function fHandleSubcribeNotification(req: Request, res: Response):
 
     // TODO: we need zendesk bot admin so that admin requests can be made by the
     // bot and not from an actiing user
-    const fakeBotContext: AppContext = {
+    const fakeBotContext: AppContextWithBot = {
+        app_id: getManifest().app_id,
         acting_user_id: 'rgixs6uimp88tq8x8w3yxu3oqe',
-    };
+    } as AppContextWithBot;
```

Comment:
There's no bot id in this context. Why is it being casted to `AppContextWithBot`?

Reply:
Actually, there should be a bot that we can use, right? See L31.

Reply:
A refactor has removed the need for this fake context variable